functional_tests.go

```diff
@@ -6244,7 +6255,7 @@ func testPutObjectWithContextV2() {
 	}
 	defer c.RemoveBucket(bucketName)
 	bufSize := thirtyThreeKiB
```

Comment:
bufSize can also be picked from the map based on filename

Reply:
done